multinode-demo/start_nodes.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Format for the nodes file
+# node1 public-ip|node1 privte-ip
+# node2 public-ip|node2 privte-ip
+# ....
+# e.g.
+# 35.197.92.75|10.138.0.6
+# 35.185.232.222|10.138.0.3
+# 35.233.137.53|10.138.0.5
+
+nodes_file=$1
+remote_user=$2
+ssh_keys=$3
+
+declare -A ip_addr_array
+while IFS=\| read public private
```

Comment:
I far prefer the one-line form for these statements for the same reason as ❤️ 1TBS, much less lines:
```bash
while IFS=\| read public private; do
```
...
```bash
for public_ip in ${!ip_addr_array[*]}; do
```
...
```bash
if [ $count -eq 0 ];  then
```

Reply:
👍